pyvrp/cpp/diversity/broken_pairs_distance.cpp

```diff
@@ -1,6 +1,9 @@
 #include "diversity.h"
 
-double brokenPairsDistance(Solution const &first, Solution const &second)
+using namespace pyvrp;
+
```

Comment:
I would rather not use `using namespace` stuff because it's somewhat similar to `from X import *`, with much the same problems. But that'd add a lot of additional lines to this PR, because then I have to spell out every type we include from other places. I might do so in a follow-up PR. 

For now I didn't use it when it was really easy to avoid it, but I did use it for long/large files (mostly in `pyvrp` and `pyvrp::search`).